README.md

```diff
@@ -101,7 +102,8 @@ Example commands:
 
```

Comment:
![image](https://github.com/BHObserver/To-Do-List/assets/93039359/c6fe3d22-6085-4927-a873-252cc1e954cf)

- [ ] It would be nice if you fix the highlighted links in the image above because right now these links are broken 😨 Kindly fix them or remove them (if you don't have any dedicated sections for these links) to make your readme descriptive and professional 👍